### PR TITLE
Rename requireConsentForCampaignTracking to disableCampaignParameters

### PR DESCRIPTION
**Pull request by caddoo**
### Description:

The documentation has changed so I've update title/desc, as well as made the config name consistent.

follow up to #766

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Review by AltamashShaikh — CHANGES_REQUESTED** [review]
@caddoo You need to update the migration [script](https://github.com/matomo-org/tag-manager/blob/5.x-dev/Updates/5.1.0-b1.php#L62) too.

**Comment by caddoo** [comment]
> @caddoo You need to update the migration [script](https://github.com/matomo-org/tag-manager/blob/5.x-dev/Updates/5.1.0-b1.php#L62) too.

Can you help me a bit, is it ok to edit an existing migration script or should I create a new one?

**Comment by AltamashShaikh** [comment]
> > @caddoo You need to update the migration [script](https://github.com/matomo-org/tag-manager/blob/5.x-dev/Updates/5.1.0-b1.php#L62) too.
> 
> Can you help me a bit, is it ok to edit an existing migration script or should I create a new one?

Its okay to edit this one, as we haven't released the new version

**Comment by AltamashShaikh** [comment]
Note for myself: After merging this PR make similar changes [here](https://github.com/matomo-org/google-tag-manager-matomo-template/pull/8) as this is a GTM Matomo template for cloud users.

File: tests/System/expected/test___TagManager.exportContainerVersion_site_default_container.xml [hunk 1]
```diff
@@ -6,19 +6,19 @@
 	<name>Default Container</name>
 	<description>This container was auto generated when the website was created.</description>
 	<ignoreGtmDataLayer>0</ignoreGtmDataLayer>
-
```
Comment (AltamashShaikh) [inline tests/System/expected/test___TagManager.exportContainerVersion_site_default_container.xml:9]: @caddoo This extra spaces are the reason for tests failing

File: lang/en.json [hunk 2]
```diff
@@ -622,10 +622,10 @@
...
         "MatomoConfigurationMatomoRequestContentTypeTitle": "Content-Type",
         "MatomoConfigurationMatomoRequestContentTypeDescription": "Set request Content-Type header value for POST requests.",
-        "MatomoConfigurationMatomoRequireConsentForCampaignTrackingTitle": "Require consent to track campaign parameters",
-        "MatomoConfigurationMatomoRequireConsentForCampaignTrackingDescription": "By enabling the consent requirement, Matomo will track campaign parameters only for users who have provided their consent. This approach ensures that data collection respects users' preferences while enabling the analysis of traffic segmentation by key metrics like source, medium, and campaign specifics.",
+        "MatomoConfigurationMatomoDisableCampaignParametersTitle": "Disable Campaign Parameters Tracking",
+        "MatomoConfigurationMatomoDisableCampaignParametersDescription": "When this option is checked, Matomo will not track campaign parameters and will be removed from the tracked URLs.",
```
Comment (michalkleiner) [inline lang/en.json]:
```suggestion
        "MatomoConfigurationMatomoDisableCampaignParametersDescription": "When this option is checked, Matomo will not track campaign parameters and they will be removed from the tracked URLs.",
```

File: tests/System/expected/test_webContext__TagManager.getAvailableVariableTypesInContext.xml [hunk 3]
```diff
@@ -1229,16 +1229,16 @@
...
+						<title>Disable Campaign Parameters Tracking</title>
 						<value>0</value>
 						<defaultValue>0</defaultValue>
 						<type>boolean</type>
 						<uiControl>checkbox</uiControl>
 						<uiControlAttributes>
 						</uiControlAttributes>
 						<availableValues />
-						<description>By enabling the consent requirement, Matomo will track campaign parameters only for users who have provided their consent. This approach ensures that data collection respects users' preferences while enabling the analysis of traffic segmentation by key metrics like source, medium, and campaign specifics.</description>
+						<description>When this option is checked, Matomo will not track campaign parameters and will be removed from the tracked URLs.</description>
```
Comment (michalkleiner) [inline tests/System/expected/test_webContext__TagManager.getAvailableVariableTypesInContext.xml]:
```suggestion
						<description>When this option is checked, Matomo will not track campaign parameters and they will be removed from the tracked URLs.</description>
```

**Review by michalkleiner — APPROVED** [review]
Looks good to me

**Review by snake14 — APPROVED** [review]
Looks good to me. No issues found in local testing :+1:

**Review by AltamashShaikh — APPROVED** [review]

**Pull request merged**